### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by ursaciuc-adrian**

### CODE REVIEW

File: FiiPrezent.Tests/EventsServiceTests.cs [hunk 1]
```diff
@@ -54,41 +55,20 @@ public void TryCreateEvent_WhenCodeIsUnused_ReturnsNoErrors()
...
-                                    x.Name == "name" && 
-                                    x.Description == "description" && 
-                                    x.VerificationCode == "code")));
-        }
 
-        [Fact]
-        public void TryCreateEvent_WhenCodeIsTooShort_ReturnsError()
+        [Theory]
+        [InlineData("name", "description", "cometothedarksidewehavecookies", ErrorType.CodeAlreadyExists)]
+        [InlineData("cometothedarksidewehavecookies", "description", "code", ErrorType.NameIsTooLong)]
```
Comment (sdamian) [inline FiiPrezent.Tests/EventsServiceTests.cs:61]: What's the name limit? Maybe add tests for the edge cases.

**Review by sdamian — CHANGES_REQUESTED** [review]
Yay conflicts... Cand am facut merge la PR-ul precedent am dat squash. Asta inseamna ca in loc de un commit pt merge si inca unul cu schimbarile tale acum pe master am un singur comit cu schimbarile (istoricul e mai curat asa). 
Problema e ca acum pe master gasesti alt commit decat cel original.. deci git-ul considera ca ai douca commit-uri noi pe qtblaze:tdd si de acolo conflictele.
Cand faci un PR cel mai sigur e sa stergi stergi branch-ul cand e gata.
Ca sa nu faci alt PR poti sa incerci `git reset --hard upstream/mater` si `git cherry-pick 3711714cc4028653b47d13ddd50e95e0413fb19b`. Dupa care `git push -f` ;)

File: FiiPrezent.Tests/EventsServiceTests.cs [hunk 2]
```diff
@@ -98,47 +78,25 @@ public void TryCreateEvent_WhenNameIsMissingAndCodeIsTooShort_ReturnsError()
...
-
-            var result = eventsService.TryCreateEvent("", "descr", "code");
-
-            result.ErrorsList.ShouldContain(ErrorType.NameIsMissing);
+                .ShouldBe(new[] { ErrorType.NameIsMissing, ErrorType.CodeIsTooShort });
         }
 
         [Fact]
-        public void TryCreateEvent_WhenNameIsTooLong_ReturnsError()
+        public void RegisterParticipant_WhenAllDataIsValid_CreateEventWithNoParticipants()
```
Comment (sdamian) [inline FiiPrezent.Tests/EventsServiceTests.cs:87]: Daca faci si testul cu `RegisterParticipant_WhenDataIsNotValid_DoesNotCreateEvent` s-ar putea sa ai o surpriza 😮

**Comment by sdamian** [comment]
merged pr #8 instead
